tests/test_pipelines.py

```diff
@@ -196,15 +196,20 @@ def test_ddim(self):
         ddpm.set_progress_bar_config(disable=None)
 
         generator = torch.manual_seed(0)
-        image = ddpm(generator=generator, num_inference_steps=2, output_type="numpy")["sample"]
+        image = ddpm(generator=generator, num_inference_steps=2, output_type="numpy").images
+
+        generator = torch.manual_seed(0)
+        image_from_tuple = ddpm(generator=generator, num_inference_steps=2, output_type="numpy", return_dict=False)[0]
 
         image_slice = image[0, -3:, -3:, -1]
+        image_from_tuple_slice = image_from_tuple[0, -3:, -3:, -1]
```

Comment:
These checks can probably be covered by `test_pipeline_outputs_equivalence`? Or are the pipeline inputs too specific for a general test?

Reply:
Haha yeah we just don't have a general pipeline tester yet - for the sake of going a bit quickly here - ok to leave it here for now and open an issue (create general PipelineTester?)